### ISSUE TITLE
`gnome-online-accounts 3.48` is incompatible with `libxml2 2.12` 

### ISSUE THREAD

**Issue by PJungkamp** [NixOS/nixpkgs#294711]
### Describe the bug

`gnome-online-accounts` crashes on startup if a Microsoft exchange account is configured. See https://gitlab.gnome.org/GNOME/libxml2/-/issues/702

### Steps To Reproduce

Register an EWS account in `gnome-control-center`.

### Expected behavior

`gnome-online-accounts` does not crash.

### Notify maintainers

`gnome-online-accounts` needs to be bumped to `3.49`.

@bobby285271
@hedning
@jtojnar
@dasj19
@amaxine

### Metadata

```console
$ nix run -- nixpkgs#nix-info -m
 - system: `"x86_64-linux"`
 - host os: `Linux 6.7.6, NixOS, 24.05 (Uakari), 24.05.20240224.73de017`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.1`
 - channels(pjungkamp): `""`
 - channels(root): `""`
 - nixpkgs: `/nix/store/gzv0nsm8d5js3wzq2mr1sv9lfp1iyail-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by amaxine** [comment]
3.49 is unstable, so that'll make it in as 3.50 with gnome 46 and not before. We probably want to pin libxml to 2.11 within goa instead, or figure out if we can cherry pick patches to fix this for 3.48

**Comment by jtojnar** [comment]
We probably cannot pin libxml since it is pretty pervasive in the closure. Cherry-picking is easy here: https://github.com/NixOS/nixpkgs/pull/294770
